contents/blog/posthog-vs-kubit.md

```diff
@@ -0,0 +1,412 @@
+---
+date: 2023-07-26
+title: In depth: PostHog vs Kubit
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+featuredImage: ../images/blog/posthog-vs-kubit/posthog-vs-kubit.jpg
+featuredImageType: full
+author:
+    - joe-martin
+category: General
+tags:
+    - Comparisons
+    - Guides
+---
+
+Kubit is a product analytics platform built for product teams. PostHog, on the other hand, is built primarily for engineers. In this article we’ll explore the differences between Kubit and PostHog in detail, but it can helpful to first discuss the different needs of engineers and product teams:  
```

Comment:
```suggestion
Kubit is a product analytics platform built for product teams. PostHog, on the other hand, is built primarily for engineers. 

In this article we’ll explore the differences between Kubit and PostHog in detail, but it can helpful to first discuss the different needs of engineers and product teams:  
```